### PR TITLE
Third deliverable

### PR DESCRIPTION
**Pull request by mjayalam**
It adds the third deliverable

### CODE REVIEW

**Comment by mjayalam** [comment]
The link to the product list page to filter by category using query params works by clicking on the categories listed on the home page

File: src/components/FeaturedProductsGrid.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import { useFetch } from "../utils/hooks/useFetch";
+import Card from "./common/Card";
+import Flex from "./common/StyledFlex";
+import ImageContainer from "./common/Image";
+
+const FeaturedProductGrid = () => {
+    const { data, isLoading } = useFetch('product',true);
```
Comment (gusgarciaw) [inline src/components/FeaturedProductsGrid.js:7]: Remember to convert this component `stateless`. You need to move this fetch to the parent component and only pass the data as props.

File: src/components/ProductCategoriesList.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import Card from './common/Card';
+import ImageContainer from './common/Image';
+import { useHistory } from 'react-router-dom';
+import { useFetch } from '../utils/hooks/useFetch';
+
+const ProductCategoriesList = () => {
+    const { data, isLoading } = useFetch('category',false,16);
```
Comment (gusgarciaw) [inline src/components/ProductCategoriesList.js:8]: Same above comment.

**Review by gusgarciaw — COMMENTED** [review]
Good Job 👍. You missed to add `PropTypes` but in general good job. I left some comments about best practices.

**Pull request merged**